src/client/components/pages/collections.js

```diff
@@ -41,9 +42,16 @@ class CollectionsPage extends React.Component {
 
 	handleTypeChange(type) {
 		const querySearchParams = type ? `type=${type}` : '';
-		this.setState({querySearchParams});
+		this.setState({querySearchParams, type});
 	}
 
+	searchParamsChangeCallback = (searchParms) => {
+		const type = searchParms.get('type') ? searchParms.get('type') : '';
```

Comment:
```suggestion
		const type = searchParms.get('type') ?? '';
```

The `??` operator means "if left side is null or undefined, return the right side", which basically means a default value.
In this case, searchParams.get will return either the parameter or `null` if there isn't one, so it's exactly appropriate (your code works, just suggesting a more succinct approach)

Reply:
?? operator would be perfect for this , thanks !